### PR TITLE
chore: better error messages

### PR DESCRIPTION
**Pull request by crenshaw-dev**

### CODE REVIEW

**Review by ashutosh16 — APPROVED** [review]
LGTM

File: util/db/cluster.go [hunk 1]
```diff
@@ -233,17 +233,17 @@ func (db *db) GetCluster(_ context.Context, server string) (*appv1.Cluster, erro
 func (db *db) GetProjectClusters(ctx context.Context, project string) ([]*appv1.Cluster, error) {
 	informer, err := db.settingsMgr.GetSecretsInformer()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to get secrets informer: %w", err)
 	}
 	secrets, err := informer.GetIndexer().ByIndex(settings.ByProjectClusterIndexer, project)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to get index by project cluster indexer: %w", err)
```
Comment (ishitasequeira) [inline util/db/cluster.go:240]: nit: Maybe add the error message to specify which project?

Comment (crenshaw-dev) [inline util/db/cluster.go:240]:
```suggestion
		return nil, fmt.Errorf("failed to get index by project cluster indexer for project %q: %w", project, err)
```

Comment (crenshaw-dev) [reply util/db/cluster.go]: done!

**Review by wanghong230 — APPROVED** [review]

**Pull request merged**